source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -356,27 +443,24 @@ def test_create_user_auto_activate(self, imailing):
             'last_name': 'Norris',
         }
 
-        instance_imailing = imailing.create_instance.return_value
-        instance_imailing.send_templated_email.return_value = {
-            "code": "failure",
-        }
-
         response = self.client.post(
             reverse('users'),
             data,
             format='json',
         )
 
         content = {
-            'detail': "The account was created but no email was "
-                      "sent. If your account is not activated, "
-                      "contact the administration.",
+            'detail': "The email service is not activated.",
```

Comment:
Idem here, customer need to know that the account has been created.

Reply:
done